arwen/host/arwen.go

```diff
@@ -262,6 +262,11 @@ func (host *vmHost) InitState() {
 	host.initContexts()
 }
 
+// Close will close all underlying processes
+func (host *vmHost) Close() {
```

Comment:
if you would have chosen to have the signature to something like
```GO
func (host *vmHost) Close() error
```
the arwen VM would have implemented `Closer` interface which is easier to integrate

Reply:
ok :). will do. And it will be simpler in elrond-go.